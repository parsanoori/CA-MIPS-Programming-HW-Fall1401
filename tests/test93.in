27
817670984
820537201
130645843
383504939
420664559
532769602
1862509
1053059536
115632475
1015269864
870660699
512220309
190267560
522804535
802578607
395348531
880258971
1019661902
300951130
422401585
584036010
1064673938
485262267
894005960
968164059
85016839
990717382