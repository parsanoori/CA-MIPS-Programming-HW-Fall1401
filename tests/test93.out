1862509
85016839
115632475
130645843
190267560
300951130
383504939
395348531
420664559
422401585
485262267
512220309
522804535
532769602
584036010
802578607
817670984
820537201
870660699
880258971
894005960
968164059
990717382
1015269864
1019661902
1053059536
1064673938